.github/CODEOWNERS

```diff
@@ -19,6 +19,7 @@
 
 # Event Ingestion
 /src/sentry/attachments/           @getsentry/owners-ingest
+/src/sentry/api/endpoints/relay/** @gestentry/owners-ingest
```

Comment:
```suggestion
/src/sentry/api/endpoints/relay/** @getsentry/owners-ingest
```

Reply:
how did that happen! i wish there was validation. 

thanks!

Reply:
Github adds squiggly lines when you open the diff, otherwise I would probably not have spotted it :)